lib/uwb/protocols/fira/UwbOobConversions.cxx

```diff
@@ -0,0 +1,126 @@
+
+#include <uwb/protocols/fira/UwbOobConversions.hxx>
+
+using namespace uwb::protocol::fira;
+/**
+ * @brief map of uci tag to a function of how to generate that uci tag given UwbConfiguration
```

Comment:
Can we say "parameter type" instead of tag to avoid confusion with the TLV tags used in the OOB structures?
```suggestion
 * @brief map of uci parameter type to a function of how to generate that parameter value given UwbConfiguration
```
Same goes for other places here where `tag` is used in the code.